Appsilon/mbaza
# Add export popover

Closes #229 

### Description
This task provides UI adjustments for #175 

### How to test
- check export popover appearance
- check functionality of each button inside
